packages/checkbox/checkbox.scss

```diff
@@ -23,9 +42,11 @@ $checkbox-check-mark-color: $checkbox-background-color--unchecked;
 
     &:active > .jkl-checkbox__check-mark {
         transform: scale(0.9);
-        background-color: $checkbox-background-color--checked;
+        background-color: $checkbox-background-color--checked; // legacy IE11 support
```

Comment:
Btw, leste interessant artikkel i helgen om (tror det var webarchives) som nå dropper støtte, og en artikkel fra Microsoft som iallefall der (ser ut til at de gikk litt tilbake på det etter artikkelen) anser IE11 som compatbilitiy mode. Så jeg lurer på om ikke vi kan gå en ny runde, så slipper vi dette.

Evt så har jeg satt opp en postcss puligin i aksept for å unngå å måtte skrive alt dette fjaset selv.

Reply:
Jeg også har sett på en annen variant i helgen der all legacy-koden er i en egen fil, så man potensielt kan bygge to versjoner. Men hadde vært sykt nice å slippe helt! Skal ta en titt i KFP på plugin-en din (husker du nevnte noe om det på demoen).

Reply:
Kjipt å fortsatt måtte støtte IE11, men enn så lenge trenger BM at IE er med på laget selv om det er som ballgutt...